ember-cli-build.js

```diff
@@ -7,7 +7,7 @@ module.exports = function (defaults) {
     // Add options here
     babel : {
       plugins : [
-        'transform-object-rest-spread',
+        [require('babel-plugin-transform-object-rest-spread').default],
```

Comment:
Can you please link to any more or less official page that suggests this as the correct way of defining Babel plugins in Ember CLI 3.1+?

Reply:
random googling, should probably not be included, will remove